javascript/javascript1/week1/Ez Namey/Ez_Namey.js

```diff
@@ -0,0 +1,5 @@
+const firstWords = ['Easy','Awesome','Real','Web','Design','Real','Tech','Snap','Trade','Info'];
+const secondWords = ['Corporate','Technology','Corporation','Solutions','Services','Manufacturing','Security','Agency','Consulting','Estate'];
+const randomNumber = Math.floor(Math.random() * 10) + 0;
```

Comment:
adding zero is redundant here. It sufficient to have `Math.floor(Math.random() * 10); `

Reply:
i have removed the 0 and committed changes